src/pretix/helpers/database.py

```diff
@@ -73,19 +74,17 @@ def __init__(self, *expressions, ordered=False, **extra):
             extra.update({'separator': ','})
         super().__init__(*expressions, **extra)
 
-    def as_postgresql(self, compiler, connection):
+    def as_postgresql(self, compiler, connection, **extra_context):
         if self.ordered:
-            return super().as_sql(
-                compiler, connection,
-                function='string_agg',
-                template="%(function)s(%(field)s::text, '%(separator)s' ORDER BY %(field)s ASC)",
-            )
+            template = "%(function)s(%(distinct)s%(field)s::text, '%(separator)s' ORDER BY %(field)s ASC)"
```

Comment:
funktioniert das bei dir?
mein postgresql akzeptiert das nur so:
```suggestion
            template = "%(function)s(%(distinct)s%(field)s::text, '%(separator)s' ORDER BY %(field)s::text ASC)"
```


ansonsten sagt es `in an aggregate with DISTINCT, ORDER BY expressions must appear in argument list`